konan/konan.properties

```diff
@@ -32,7 +32,7 @@ downloadingAttempts = 10
 downloadingAttemptIntervalMs = 3000
 homeDependencyCache = .konan/cache
 
-llvmDebugOptFlags = -O1
+clangDebugFlags = -O0
```

Comment:
So what is the current binary size?

Reply:
A bit bigger for debug build. On macOS it's 1.3M vs 1.0M.